api.c

```diff
@@ -572,10 +571,6 @@ int tcmu_emulate_evpd_inquiry(
 		/* Optimal xfer length */
 		memcpy(&data[12], &val32, 4);
 
-		/* MAXIMUM WRITE SAME LENGTH */
-		val64 = htobe64(VPD_MAX_WRITE_SAME_LENGTH);
-		memcpy(&data[36], &val64, 8);
-
```

Comment:
Full size like 0xFFFF FFFF ？ or just like the linux do by setting it to 8M(0x7FFFFF)?